core/src/main/java/org/apache/accumulo/core/file/rfile/RFileOperations.java

```diff
@@ -128,9 +129,9 @@ protected FileSKVWriter openWriter(OpenWriterOperation options) throws IOExcepti
       outputStream = fs.create(new Path(file), false, bufferSize, (short) rep, block);
     }
 
-    CachableBlockFile.Writer _cbw = new CachableBlockFile.Writer(
+    BCFile.Writer _cbw = new BCFile.Writer(
         new RateLimitedOutputStream(outputStream, options.getRateLimiter()), compression, conf,
-        acuconf);
+        false, acuconf);
```

Comment:
What does this false parameter accomplish? Can we do it with an enum instead of a boolean?